data/query_test.go

```diff
@@ -21,12 +21,14 @@ func TestUnitReviewQuerySuccess(t *testing.T) {
 		So(err, ShouldBeNil)
 
 		urlQuery := url.Values{
-			"q":      []string{"housing"},
-			"filter": []string{"article"},
-			"topics": []string{"1234,5678"},
-			"sort":   []string{"relevance"},
-			"limit":  []string{"10"},
-			"page":   []string{"1"},
+			"q":                []string{"housing"},
+			"population_types": []string{""},
+			"dimensions":       []string{""},
+			"filter":           []string{"article"},
+			"topics":           []string{"1234,5678"},
+			"sort":             []string{"relevance"},
+			"limit":            []string{"10"},
+			"page":             []string{"1"},
```

Comment:
Have you added any query tests for the new params?

Reply:
There's not much to test, but added a check that it's the same after the map function.